sensing/autoware_pointcloud_preprocessor/src/downsample_filter/voxel_grid_downsample_filter_node.cpp

```diff
@@ -66,9 +66,9 @@ VoxelGridDownsampleFilterComponent::VoxelGridDownsampleFilterComponent(
 {
   // set initial parameters
   {
-    voxel_size_x_ = static_cast<float>(declare_parameter("voxel_size_x", 0.3));
```

Comment:
In the past, declaring parameters as `float` was not possible.
Are you sure it is ok now?

For reference:
https://github.com/ros2/rclcpp/blob/e6b6faf152ae3c758f73a35ad1385bfd987f346f/rclcpp/include/rclcpp/parameter_value.hpp#L35

Reply:
update: they did not change the old structure but added other stl tricks to add compatibility

Reply:
Thanks!!